GoogleForCreators/web-stories-wp
# Make WAF workaround opt-in for now

## Summary

Turns out the mbstring polyfill we include is UTF-8 centric and not suitable for the UTF-16 conversion we are doing. That's why this decoding currently silently fails (i.e. leaves the string unchanged) when the native mbstring extension is not available (which can be the case on some systems).

For this reason, I am putting this behind a feature flag for now until this edge case is addressed.

Note: the mbstring polyfill is still needed because we need UTF-8 conversion for the AMP output on the frontend.

## Relevant Technical Choices



## To-do

* [ ] Remove flag once stable

## User-facing changes

N/A

## Testing Instructions

Verify that _encoding_ only happens when feature flag is turned on.

The _decoding_ always happens.

---



See #4805
